AutomaticWorkAssignment/Source/WorkSpecification.cs

```diff
@@ -19,6 +19,7 @@ public class WorkSpecification : IExposable, ILoadReferenceable
         public bool RequireFullPawnCapability = true; // Job will not be assigned if a pawn is unable to do some of the work. If off, pawn must only be able to do at least one thing.
         public bool InterweavePriorities; // Subsequent assignments to pawns will be shifted right.
         public bool IsSpecialist; // Job will prohibit assignments to jobs further down the list.
+        public bool IsIgnoreSpecialist = false; // The job tries to be assigned according to the rules, even if the pawn was assigned to the job by a specialist
```

Comment:
```suggestion
        public bool IsIgnoreSpecialistFlag = false; // The job tries to be assigned according to the rules, even if the pawn was assigned to the job by a specialist
```

or

```suggestion
        public bool IsIncludeSpecialist = false; // The job tries to be assigned according to the rules, even if the pawn was assigned to the job by a specialist
```

Seems clearer to me: `IsIgnoreSpecialist` can be understood as "Do not consider pawn marked as specialist" yet you're doing the opposite: consider pawns even if they may be specialists

Reply:
I like "IsIncludeSpecialist". I'll fix it after the tests